### PR TITLE
necessary files added for first version of Turiyan GUI

### PR DESCRIPTION
**Pull request by pankajssp**

### CODE REVIEW

**Comment by pankajssp** [comment]
Hope, now its all fine.

File: rasmalai/urls.py [hunk 1]
```diff
@@ -17,6 +17,6 @@
 from django.contrib import admin
 
 urlpatterns = [
-    url(r'^register/', include('account.urls')),
+    url(r'^turiyan/', include('account.urls')),
```
Comment (koushikkm) [inline rasmalai/urls.py]: Let this be register only and not turiyan. Acutally, link to register should be available in home page. After clicking that we get the register view executed which will display the form.

**Comment by koushikkm** [comment]
except for minor comment, rest looks good :+1:

File: rasmalai/urls.py (hunk 1 above)
Comment (pankajssp) [inline rasmalai/urls.py]: Okay. view taken into consideration.

**Comment by koushikkm** [comment]
@pankajssp : please do NOT close this pull request. You need to merge it, otherwise changes will not come to master repo. I am reopening the request again, you please merge it, look for the merge link.

**Pull request merged**